apis/nucleus/src/components/listbox/useConfirmUnfocus.jsx

```diff
@@ -0,0 +1,39 @@
+import { useRef, useEffect, useState } from 'react';
+
+/**
+ * Hook that confirms selection when
+ * user interact outside of passed element ref.
+ *
+ * @param ref - Ref to HTMLelement
+ * @param selections
+ */
```

Comment:
This would get added ot the public API, I don't think we want that?

Reply:
That would be unnecessary, will remove. Thanks for input!